### PR TITLE
Use @PHP71Migration rules

### PR DESCRIPTION
**Pull request by kubawerlos**
Ruleset `@PHP71Migration:risky` is already used, `@PHP71Migration` should be too, right?

### CODE REVIEW

File: src/Fixer/Naming/NoHomoglyphNamesFixer.php [hunk 1]
```diff
@@ -233,9 +233,8 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
             }
 
             $replaced = Preg::replaceCallback('/[^[:ascii:]]/u', static function (array $matches) {
-                return isset(self::$replacements[$matches[0]])
-                    ? self::$replacements[$matches[0]]
-                    : $matches[0]
+                return self::$replacements[$matches[0]]
```
Comment (julienfalque) [inline src/Fixer/Naming/NoHomoglyphNamesFixer.php:236]: Maybe we can make this statement single line?

**Review by julienfalque — APPROVED** [review]

File: src/Fixer/Naming/NoHomoglyphNamesFixer.php [hunk 2]
```diff
@@ -233,9 +233,8 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
...
+                    ?? $matches[0]
```
Comment (keradus) [inline src/Fixer/Naming/NoHomoglyphNamesFixer.php:237]:
```suggestion
                return self::$replacements[$matches[0]] ?? $matches[0]
```

File: src/Fixer/Naming/NoHomoglyphNamesFixer.php (hunk 1 above)
Comment (keradus) [reply src/Fixer/Naming/NoHomoglyphNamesFixer.php]: done

**Review by keradus — APPROVED** [review]

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**